js/src/utils.ts

```diff
@@ -206,6 +206,43 @@ export async function getAllDomains(
   return accounts.map((a) => a.pubkey);
 }
 
+/**
+ * This function can be used to retrieve all domain names owned by `wallet` in a human readable format
+ * @param connection The Solana RPC connection object
+ * @param wallet The wallet you want to search domain names for
+ * @returns Array of pubkeys and the corresponding human readable domain names
+ */
+export async function getAllReversedDomains(
```

Comment:
Honestly I don't like the naming here, it doesn't describe to me what it really does. I mean the `reversed` part. WDYT about something like `getVerboseDomains`? Or something that describes it better. I'm not a native speaker tho so in both my concern and my suggestion I might be wrong :))

Reply:
No it's totally valid. How about something like, getDomainStrings?

Reply:
`getDomainKeysWithReverses`, `getDomainKeysAndReverses`?

Reply:
I think getDomainKeysWithReverses is good. Updating now